examples/stable_diffusion_v2/schedulers.md

```diff
@@ -35,24 +35,27 @@ You can obtain diverse results according to the given **prompt**. Here are 5 exa
 | :----: | :----: | :----: | :----: | :----: |
 | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/6a84e5ae-baab-4461-91ec-14f55b16aee2" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/62c0bb93-68a8-4ff1-8bc5-0c9305931ac0" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/92441896-8786-465b-82a2-3438aad85476" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/e626ae1b-8fa6-4ad8-a083-ec39e4944277" width="155" height="155" /> | <img src="https://github.com/zhaoyuzhi/mindone/assets/13333802/d3728d81-4a31-4d53-baf0-cd79b6046ca3" width="155" height="155" /> |
 
-### Fast Inference
+### Changing the scheduler
 
-For fast inference, we provide several scripts using different schedulers:
+As is known, it is extremely difficult to measure quantitatively which scheduler works best for a given diffusion pipeline, so it is often recommended to simply try out which works best. You can test the stable diffusion model with different scheduler by the following
+commands. In addition to the default scheduler **plms**, the **${scheduler_name}** can be one of **ddim**, **dpm_solver**, **dpm_solver_pp** and **uni_pc** .
 
-```bash
-# Text to image generation with SD2.0, using PLMS scheduler
-bash scripts/tests/test_plms_sampler.sh
-# Text to image generation with SD2.0, using DDIM scheduler
-bash scripts/tests/test_ddim_sampler.sh
-# Text to image generation with SD2.0, using DPM-Solver scheduler
-bash scripts/tests/test_dpmsolver_sampler.sh
-# Text to image generation with SD2.0, using DPM-Solver++ scheduler
-bash scripts/tests/test_dpmsolverpp_sampler.sh
-# Text to image generation with SD2.0, using UniPC scheduler
-bash scripts/tests/test_unipc_sampler.sh
+```shell
+# Text to image generation with SD2.0. Select one ${scheduler_name}, otherwise the scheduler will be plms.
+python text_to_image.py \
+    --prompt "A Van Gogh style oil painting of sunflower" \
+    --config configs/v2-inference.yaml \
+    --output_path ./output/ \
+    --seed 42 \
+    --${scheduler_name}
+    --n_iter 8 \
+    --n_samples 1 \
+    --W 512 \
+    --H 512 \
+    --sampling_steps 50
 ```
 
-The default prompt is **A Van Gogh style oil painting of sunflower**. Some hyper-parameters are not shared for different schedulers, e.g., sampling steps. For instance, DPM-Solver, DPM-Solver++, and UniPC need smaller sampling steps (e.g., 20) than DDIM and PLMS samplers (e.g., 50).
+The default prompt is **A Van Gogh style oil painting of sunflower**. Some hyper-parameters are not shared for different schedulers, e.g., **sampling steps**. For instance, DPM-Solver, DPM-Solver++, and UniPC need smaller sampling steps (e.g., 20) than DDIM and PLMS samplers (e.g., 50).
```

Comment:
"Some hyper-parameters are not shared for different schedulers"

The optimal hyper-parameters can vary for different schedulers, e.g., the optimal `sampling_steps` is 50 for PLMS, and 20 for UniPC.